tests/unit/test_helpers.py

```diff
@@ -301,5 +307,14 @@ def test_prepare_form_post(self):
                 method='POST'):
             self.assertEqual(prepare_form(flask.request), json_data)
 
+    def test_json_array(self):
+        """Check if we return the right type of each JSON."""
+        # A simple JSON array
```

Comment:
Do we mean object instead of array?

Reply:
And I meant an `object`. doh! And here goes all my theatrical makeup over my face. Thanks.